### PR TITLE
create Components for Header  and show them in Storybook

### PR DESCRIPTION
**Pull request by jbaltres**
Create the design for the Header.

image

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+function Header() {
+  return (
+    <Title>
+      <Container>
+        <Logo />
+      </Container>
+      <Headline>EventListener</Headline>
+      <Menu></Menu>
+    </Title>
```
Comment (marwinburesch) [inline src/components/Header.js]: <Title> seems to function as a wrapper for the Header component, you could name it accordingly, and use <Title> where <Headline> is right now. Also, <Container> is a little too generic in my opinion. "LogoWrapper" or "LogoContainer" would be bit more descriptive.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+import Logo from "../icons/Logo.js";
+
+const Title = styled.header`
+  height: 80px;
+  width: 375px;
```
Comment (marwinburesch) [inline src/components/Header.js]: Are you sure you want to use a fixed width for the parent container?

Comment (jbaltres) [reply src/components/Header.js]: In the project I wont use a fixed width, I just used it here to see how it looks like

File: src/components/Header.js (hunk 1 above)
Comment (jbaltres) [reply src/components/Header.js]: Thx for the comment, i will rename the Component.

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  margin: 0;
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-image: linear-gradient(to right, rgba(37, 206, 197, 1), black) 50 15%;
+`;
+
+const Container = styled.div`
+  align-items: center;
+  margin: 15px 15px;
```
Comment (lmachens) [inline src/components/Header.js]: same like `margin: 15px`

File: src/stories/Logo.stories.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Logo from "../icons/Logo";
+
+export default {
+  title: "Logo"
+};
+
+export const DefaultLogo = () => <Logo></Logo>;
```
Comment (lmachens) [inline src/stories/Logo.stories.js]: `<Logo></Logo>` = `<Logo />`

File: src/components/Header.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  background-color: rgb(0, 0, 0);
+  margin: 0;
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-image: linear-gradient(to right, rgba(37, 206, 197, 1), black) 50 15%;
+`;
+
+const Container = styled.div`
+  align-items: center;
```
Comment (lmachens) [inline src/components/Header.js]: are you sure `align-items` and `flex-direction` has any effect? 
this `div` is not a flexbox.

I think you don't need this container at all. 
Instead, you could style the `Logo` and apply `margin` there:

```
const StyledLogo = styled(Logo)`
  margin: 15px;
`
```

File: src/components/Header.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+const Menu = styled.img``;
+
+function Header() {
+  return (
+    <Title>
+      <Container>
+        <Logo />
+      </Container>
+      <Headline>EventListener</Headline>
+      <Menu></Menu>
```
Comment (lmachens) [inline src/components/Header.js]: `<Menu>` has no effect I think. Try to avoid adding empty components.

**Pull request merged**